package.json

```diff
@@ -4,7 +4,6 @@
   "description": "polkawallet bridge sdk",
   "main": "build/src/index.js",
   "typings": "build/src/index.d.ts",
-  "author": "Polkawallet developers <hello@polkawallet.io>",
```

Comment:
@dom had some guidance for this:
https://www.notion.so/Licensing-and-Forking-Code-2f74bea9c577458cb449a9ca59c815ec

Reply:
> @dom had some guidance for this
Good shout, thanks.
We cannot add more people to the `author` field, as it only holds a singular entity.

I'll undo this for now, to be clarified if we want to change to using a `contributors` field instead.

Reply:
With regards to the author vs contributors field, see also this [npm doc](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors)

Reply:
I think it best to replace `authors` with `contributors` and use an array with the Polkawallet developers and us in there.